internal/voting/sql.go

```diff
@@ -51,15 +51,7 @@ const (
 		id BYTEA PRIMARY KEY
 	);`
 
-	tableHeight = `CREATE TABLE IF NOT EXISTS ` + votingSchemaName + `.height (
```

Comment:
We should not delete the table here?

Reply:
Ah, yes.  Readded so it actually gets deleted on init. :)

Testing it out again.